generate-sandpack-config.mjs

```diff
@@ -0,0 +1,118 @@
+import _ from 'lodash';
+
+import fs from 'fs';
+import path from 'path';
+
+const examples = ['banner'];
+
+console.log('Sandpack config factory 🏭');
+
+examples.forEach(example => {
+  let __dirname = path.join(example);
+
+  const framworks = fs.readdirSync(__dirname);
+
+  framworks.map(framwork => path.join(__dirname, framwork)).map(createConfig);
+});
+
+function createConfig(dirname) {
+  let __dirname = path.join(dirname);
+
+  const mapFileNameToCode = mapFileNameToCodeFactory(__dirname);
+  const readAllFilesFromBlob = readAllFilesFromBlobFactory(mapFileNameToCode);
+
+  try {
+    const config = fs.readFileSync(
+      path.join(__dirname, 'sandpack.config.json'),
+      'utf8'
+    );
+
+    const packageJson = fs.readFileSync(
+      path.join(__dirname, 'package.json'),
+      'utf8'
+    );
+
+    const parsedPackage = JSON.parse(packageJson);
+    const parsedConfig = JSON.parse(config);
+
+    const { files } = parsedConfig;
+
+    let expandedFiles = [];
+
+    if (_.isArray(files)) {
```

Comment:
prefer native?

Reply:
because i'm using isPlainObject might aswell make use of isArray